gcbdr/backupplan/main.tf

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
For ease of reading, please rename each of these examples with underscore separated directory names (`backup_plan/main.tf`, `backup_plan_association/main.tf`, etc. )

Reply:
Update PR to reflect this